src/chart/Bubble.js

```diff
@@ -48,13 +66,22 @@
             .attr("class", "node")
             .attr("opacity", 0)
             .on("click", function (d) {
-                context.click(context.rowToObj(d), context._columns[1]);
+                    console.log("data", context._data);
```

Comment:
Or is it a "ITooltip" - then any chart which want to implement an ITooltip would be forced to honor the contract of "mouseover/out/move"?